cliutil/utils.go

```diff
@@ -99,26 +99,42 @@ func GetVersion() string {
 		return "unknown-version"
 	}
 
-	commitHash := "???????"
```

Comment:
We didn't have any release tags when I wrote this and I didn't know that the commit hash wouldn't be present if doing "go install" and passing the remote repository.